controllers/storageclusterpeer/storageclusterpeer_controller.go

```diff
@@ -27,6 +27,10 @@ import (
 	v1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
 )
 
+const (
+	ClientMappingConfigMapName = "client-mapping-config"
```

Comment:
'Client' and 'Mapping' are very generic terms when you consider all of ODF and not just RDR. Please choose a better name for the config map and the var